ooniprobe/AppDelegate.mm

```diff
@@ -79,54 +78,17 @@ -(void)handleNotification:(NSDictionary*)userInfo :(UIApplication *)application{
     UIApplicationState state = [application applicationState];
     if (state == UIApplicationStateActive)
     {
-        if ([type isEqualToString:@"open_href"]){
-            UIAlertAction* okButton = [UIAlertAction
-                                       actionWithTitle:NSLocalizedString(@"Modal.OK", nil)
-                                       style:UIAlertActionStyleDefault
-                                       handler:^(UIAlertAction * action) {
-                                           [self openBrowser];
-                                       }];
-            links = [[NSMutableArray alloc] init];
-            [links addObject:[[userInfo objectForKey:@"payload"] objectForKey:@"href"]];
-            if ([[userInfo objectForKey:@"payload"] objectForKey:@"alt_hrefs"]){
-                NSArray *alt_href = [[userInfo objectForKey:@"payload"] objectForKey:@"alt_hrefs"];
-                [links addObjectsFromArray:alt_href];
-            }
             [MessageUtility alertWithTitle:nil
                                    message:[NSString stringWithFormat:@"%@",[[userInfo objectForKey:@"aps"] objectForKey:@"alert"]]
-                                  okButton:okButton
                                     inView:self.window.rootViewController];
```

Comment:
what is this alert doing?

Reply:
Display the notification if the app is running